tests/Rules/Properties/data/missing-readonly-property-assign-phpdoc.php

```diff
@@ -0,0 +1,73 @@
+<?php // lint >= 7.4
+
+namespace MissingReadOnlyPropertyAssignPhpDoc;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ */
+class EntityWithAGeneratedId
+{
+
+	/**
+	 * @ORM\Id
+	 * @ORM\GeneratedValue
+	 * @ORM\Column
+	 * @readonly
+	 */
+	private int $id; // ok, ID is generated
+
+	/**
+	 * @ORM\Column
+	 * @readonly
+	 */
+	private int $unassigned;
+
+	/**
+	 * @ORM\Column
+	 * @readonly
+	 */
+	private int $doubleAssigned;
+
+	public function __construct(int $doubleAssigned)
+	{
+		$this->doubleAssigned = $doubleAssigned;
```

Comment:
I just got an idea for a rule improvement in phpstan/src. If an extension says `isInitialized=true` about a property, even the first assignment should say "readonly property is already assigned" :)